src/Github/AppendingCreateReleaseTextAggregate.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Laminas\AutomaticReleases\Github;
+
+use Laminas\AutomaticReleases\Git\Value\BranchName;
+use Laminas\AutomaticReleases\Git\Value\SemVerVersion;
+use Laminas\AutomaticReleases\Github\Api\GraphQL\Query\GetMilestoneChangelog\Response\Milestone;
+use Laminas\AutomaticReleases\Github\Value\RepositoryName;
+use Webmozart\Assert\Assert;
+
+use function array_filter;
+use function array_map;
+use function implode;
+
+final class AppendingCreateReleaseTextAggregate implements CreateReleaseText
```

Comment:
Mentioned above: the naming is a bit weird, but that's because it follows the pattern rather than the actual intent.

Something like `ConcatenateMultipleReleaseTexts` would suffice